diff --git a/include/verilatedos.h b/include/verilatedos.h
index 1d3b0c90e..e4c986b06 100644
--- a/include/verilatedos.h
+++ b/include/verilatedos.h
@@ -575,7 +575,8 @@ static inline double VL_ROUND(double n) {
 #elif defined(VL_IGNORE_UNKNOWN_ARCH)
 # define VL_CPU_RELAX()
 #else
-# error "Missing VL_CPU_RELAX() definition."
+# define VL_CPU_RELAX()
+// # error "Missing VL_CPU_RELAX() definition."
 #endif
 
 //=========================================================================
